kindle.py

```diff
@@ -219,13 +210,18 @@ def download_books(self):
         default=100,
         help="recover-index if download failed",
     )
+    parser.add_argument(
+        "-o", "--outdir", default=DEFAULT_OUT_DIR, help="dwonload output dir"
+    )
     options = parser.parse_args()
+
+    if not os.path.exists(options.outdir):
+        os.makedirs(options.outdir)
```

Comment:
I've move it here, making outdir a var that can be changed.

Reply:
cool thanks.